internal/app/grpcclient/client.go

```diff
@@ -163,10 +163,10 @@ func invoke(ctx context.Context, req *v1.ClientCompatRequest) (*v1.ClientRespons
 		return nil, err
 	}
 
-	switch req.Service {
+	switch *req.Service {
```

Comment:
This is where the generated accessors are usually used.
```suggestion
	switch req.GetService() {
```